### PR TITLE
Troubleshooting the build

### PR DESCRIPTION
**Pull request by calebhailey**
WIP changes to get the build working.

### CODE REVIEW

File: main.go [hunk 1]
```diff
@@ -96,21 +96,21 @@ func procMeminfoMetrics() {
...
 }
 
 func flushMetrics() {
 	metrics = []string{}
 }
 
-func PrintMetrics(prefix string) {
-	parseProcStat()
+func main() {
+	// parseProcStat()
```
Comment (calebhailey) [inline main.go]: I can build the plugin with this line uncommented, but the plugin will fail to run with the following output: 

```
$ ./system-profile-linux
panic: runtime error: index out of range

goroutine 1 [running]:
main.parseProcStat()
        /home/calebhailey/go/src/github.com/sensu-skunkworks/system-profile-linux/main.go:24 +0x2d0
main.main()
        /home/calebhailey/go/src/github.com/sensu-skunkworks/system-profile-linux/main.go:108 +0x34
```

File: main.go [hunk 2]
```diff
@@ -96,21 +96,21 @@ func procMeminfoMetrics() {
 func addMetric(metricType []string, value string) {
 	metricName := strings.Join(metricType, ".")
 	timeNow := time.Now().Unix()
-	outputs := []string{metricName, value, strconv.FormatInt(timeNow, 16)}
+	outputs := []string{metricName, value, strconv.FormatInt(timeNow, 10)}
```
Comment (calebhailey) [inline main.go:99]: We need decimal output, not hexidecimal.

File: main.go (hunk 1 above)
Comment (calebhailey) [reply main.go]: For context, I'm running this check inside of a `debian:stretch` Docker container. Here is an example output of my `/proc/stat` file: 

```
$ cat /proc/stat
cpu  234194 64 447634 8540690 310876 0 12635 302457 0 0
cpu0 59771 17 108123 2166023 71105 0 3091 73643 0 0
cpu1 59266 16 110469 2163287 73479 0 2697 71944 0 0
cpu2 56885 16 118344 2052338 90159 0 4331 85380 0 0
cpu3 58271 14 110696 2159041 76132 0 2514 71489 0 0
intr 63284757 220 0 0 0 3 1365 8110277 18 0 58335 92687 37053 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0
ctxt 129502867
btime 1553117025
processes 152503
procs_running 3
procs_blocked 0
softirq 30912639 0 13632581 3360 2116467 0 0 545 6543033 0 8616653
```

**Comment by calebhailey** [comment]
@sreejita-biswas can you take a look at these changes and my comments and incorporate them into a new release, tagged as `0.2`? Thanks!

**Comment by calebhailey** [comment]
@sreejita-biswas please feel free to use this PR, or re-implement these changes in master. 

Thanks!

File: main.go (hunk 2 above)
Comment (sreejita-biswas) [reply main.go:99]: Checked in all changes.

Comment (calebhailey) [reply main.go:99]: @sreejita-biswas thank you!

**Comment by calebhailey** [comment]
Build is looking good! I'm going to publish this one. 👍 

Thanks for your help @sreejita-biswas!

**Pull request merged**